src/app/payment-hub/transactions/incoming/incoming-transactions.component.ts

```
@@ -279,10 +429,39 @@ export class IncomingTransactionsComponent implements OnInit, AfterViewInit {
    * Loads a page of transactions.
    */
   loadTransactionsPage() {
+    const startTime = performance.now();
+
     if (!this.sort.direction) {
       delete this.sort.active;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)